docs/developer-docs/latest/developer-resources/content-api/integrations/php.md

```diff
@@ -0,0 +1,295 @@
+    }
+  ]
+}]
+```
+### Example
+
+```php
+<?php
+function getRestaurants(){
+    $curl = curl_init(); //Initializes curl
+    curl_setopt($curl, CURLOPT_URL, 'http://localhost:1337/restaurants');
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+    curl_setopt($curl, CURLOPT_HTTPHEADER, [
+        'Content-Type: application/json'
+    ]); // Sets header information for authenticated requests
+
+    $res = curl_exec($curl);
+    curl_close($curl);
+    print_r($res);
+}
+
+getRestaurants();
+
+```
+
+### POST Request your collection type
+
+Execute a `POST` request on the `restaurant` Collection Type in order to create a restaurant.
+
+Be sure that you activated the `create` permission for the `restaurant` Collection Type and the `find` permission for the `category` Collection type.
+
+_Request_
+
+```php
+$restaurants = array(
+  'name' => 'Calabar Kitchen',
+  'description' => 'Omo, this is a place varieties soup with catfish🦈',
+   'categories' => [2]
+);
+
+// Initializes a new cURL session
+$curl = curl_init();
+
+curl_setopt($curl, CURLOPT_URL, 'http://localhost:1337/restaurants');
+
+curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+
+// Set the CURLOPT_POST for POST request
+curl_setopt($curl, CURLOPT_POST, true);
+```
+
+Running the PHP file on the browser  will give you this response:
+
+_Response_
+
+```json
+[{
+    "id": 2,
+    "name": "Calabar Kitchen",
+    "description": "Omo, this is a place that varieties of soup with catfish🦈",
+    "published_at": "2021-03-14T11:23:13.251Z",
+    "created_at": "2021-03-14T11:23:13.260Z",
+    "updated_at": "2021-03-14T11:23:13.260Z",
+    "categories": [{
+        "id": 2,
+        "name": "Brunch ",
+        "published_at": "2021-03-14T09:22:58.909Z",
+        "created_at": "2021-03-14T09:22:51.893Z",
+        "updated_at": "2021-03-14T09:22:58.945Z"
+        }
+      ]
+}]
+```
+
+### Example
+
+```php
+
+<?php
+function getRestaurants(){
+    $curl = curl_init(); //Initializes curl
+    curl_setopt($curl, CURLOPT_URL, 'http://localhost:1337/restaurants');
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+    curl_setopt($curl, CURLOPT_HTTPHEADER, [
+        'Content-Type: application/json'
+    ]); // Sets header information for authenticated requests
+
+    $res = curl_exec($curl);
+    curl_close($curl);
+
+    print_r($res);
+}
+
+function postRestaurant(){
+    $restaurants = array(
+        'name' => 'Calabar Kitchen',
+        'description' => 'Omo, this is a place that varieties of soup with catfish🦈',
+         'categories' => [2]
+      );
+      
+      // Initializes a new cURL session
+      $curl = curl_init();
+      
+      curl_setopt($curl, CURLOPT_URL, 'http://localhost:1337/restaurants');
+      
+      curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+      
+      // Set the CURLOPT_POST for POST request
+      curl_setopt($curl, CURLOPT_POST, true);
+      curl_setopt($curl, CURLOPT_POSTFIELDS,  json_encode($data));
```

Comment:
```suggestion
      curl_setopt($curl, CURLOPT_POSTFIELDS,  json_encode($restaurants));
```

You declare and initialize the `$restaurants` variable but then try to post the undefined `$data` variable, which results in this error when performing the request:
```json
{"statusCode":400,"error":"Bad Request","message":"invalid JSON, only supports object and array"}
```
😉
```